README.md

```diff
@@ -12,3 +12,9 @@ $ pip install git+https://github.com/Chia-Network/clvm@use_clvm_rs
 Note that for now, you must use the `use_clvm_rs` branch of `clvm`.
 
 The rust code replaces `run_program` and `CLVMObject`.
+
+In order to run the unit tests, one has to pass `--no-default-features` to `cargo test`:
+
+```
+cargo test --no-default-features
+```
```

Comment:
Wow, how'd you figure this out?!

Reply:
other people had the same issue with pyo3